src/pages/groupComparison/GroupComparisonStore.ts

```diff
@@ -314,6 +321,225 @@ export default class GroupComparisonStore extends ComparisonStore {
         },
     });
 
+    readonly mutations = remoteData({
+        await: () => [this.samples, this.mutationEnrichmentProfiles],
+        invoke: async () => {
+            if (
+                this.samples.result!.length === 0 ||
+                this.mutationEnrichmentProfiles.result!.length === 0 ||
```

Comment:
we need to verify that this is the right set of profiles. @onursumer can you comment?  why is "enrichmentProfiles"? Should there ever be more than one of these?

Reply:
I think it's called enrichment profiles because the data is mainly used for enrichment in the genomic alterations tab. There can be more than one of these depending on the number of studies, because we are fetching profiles for all studies https://github.com/cBioPortal/cbioportal-frontend/blob/f99379744fe3453a8dbaaed0cd8d97d3e4e362c3/src/shared/lib/comparison/ComparisonStore.ts#L2086